### ISSUE TITLE
2.1.1 Update WdG

### ISSUE THREAD

**Issue by rzmoz** [BSData/The-9th-Age#452]

**Comment by rzmoz** [comment]
@DarkSkyT9A did you come to a conclusion on the version for WdG? Then I'll take this task

**Comment by DarkSkyT9A** [comment]
I don't really like going backwards, but I think we should align to the official version so use 2.1.5 I think. Do you agree?

**Comment by rzmoz** [comment]
Yes, that's my recommendation. We keep data files in sync regardless of content.
